.github/pull_request_template.md

```diff
@@ -1,5 +1,7 @@
-<!-- Thank for your PR! If it's your first time contributing to Taichi, please make sure you have checked out [Contributor Guideline](https://taichi.readthedocs.io/en/latest/contributor_guide.html) (last update: Feb 18, 2019). A few simple rules are mentioned there to make us work together more efficiently :-) -->
+<!-- Thank for your PR! If it's your first time contributing to Taichi, please make sure you have read Contributor Guideline(https://taichi.readthedocs.io/en/latest/contributor_guide.html) (last update: March 26, 2019). -->
 
-Related issue id = ... (if any)
+<!-- Please always prepend your PR title with tags such as [Metal], [CUDA], [Doc], [Example]. Use a lowercased tag (e.g. [cuda]), for PRs that are invisible to end-users (e.g. intermediate implementation). More details: http://taichi.readthedocs.io/en/latest/contributor_guide.html#pr-title-tags -->
```

Comment:
Copying most contents that already exist in contguide seems an overkill. I think simply  `e.g. [doc], see http://blahblah` is just enough.. btw, what is an end-user?

Reply:
I made this suggestion because it’s one less level of indirection for contributors to know immediately what format they should follow. I doubt if all contributors would read the guide if they are just sending small PRs (e.g. fixing docs).

End users are just regular Taichi users that don’t involve much in the development.